specs/light_client/merkle_proofs.md

```diff
@@ -96,36 +133,43 @@ def chunk_count(typ: SSZType) -> int:
         return len(typ.get_fields())
     else:
         raise Exception(f"Type not supported: {typ}")
+```
 
-
-def get_item_position(typ: SSZType, index: Union[int, str]) -> Tuple[int, int, int]:
+```python
+def get_item_position(typ: SSZType, index_or_variable_name: Union[int, SSZVariableName]) -> Tuple[int, int, int]:
```

Comment:
@protolambda @vbuterin 
addressing https://github.com/ethereum/eth2.0-specs/pull/1186#discussion_r314088781 in this function with `index_or_variable_name: Union[int, SSZVariableName]`. What do you think about it?

Reply:
The one issue is that I think the code will look terrible if we have to write `get_generalized_index(FooType, [SSZVariableName("bar"), 9, SSZVariableName("baz")])` instead of `get_generalized_index(FooType, ["bar", 9, "baz"])`. If we can still do the latter then I think it's good.

Reply:
Good point!

I changed the definition of `SSZVariableName` to just an alias of `str` (`SSZVariableName = str`). So we can still write `get_generalized_index(FooType, ["bar", 9, "baz"])`.